### PR TITLE
DOC add changelog to docs

### PR DESCRIPTION
**Pull request by adrinjalali**
This is the initial changelog, we don't have to be detailed here since it's the first release.

cc @skops-dev/maintainers

### CODE REVIEW

File: docs/changes.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+This is the first release of the library. It include two main modules:
+
+- ``skops.hub_utils``: tools to create a model repository to be stored on
+  `Hugging Face Hub <https://hf.co/models>`__, mainly through
+  ``skops.hub_utils.init`` and ``skops.hub_utils.push``.
+- ``skops.card``: tools to create a model card explaining what the model does
+  and how it should be used. The model card can then be stored as the
+  ``README.md`` file on the Hugging Face Hub, with pre-populated metadata to
+  help Hub understand the model.
```
Comment (BenjaminBossan) [inline docs/changes.rst:21]:
```suggestion
- :mod:`skops.hub_utils`: tools to create a model repository to be stored on
  `Hugging Face Hub <https://hf.co/models>`__, mainly through
  :func:`skops.hub_utils.init` and :func:`skops.hub_utils.push`.
- :mod:`skops.card`: tools to create a model card explaining what the model does
  and how it should be used. The model card can then be stored as the
  ``README.md`` file on the Hugging Face Hub, with pre-populated metadata to
  help Hub understand the model.
```

**Review by BenjaminBossan — COMMENTED** [review]
Looks good. Do we expect contributors to add to the changes.rst or do we collect all changes ourselves before the release? If the former, we should also amend the CONTRIBUTING.rst.

Do we want to later differentiate between kinds of changes, e.g. features vs bugfix vs breaking changes etc.?

**Comment by adrinjalali** [comment]
> If the former, we should also amend the CONTRIBUTING.rst.

Added authors, refinements to the contributing guide is coming in a separate PR.

> Looks good. Do we expect contributors to add to the changes.rst or do we collect all changes ourselves before the release?

I also rather each PR add their line in the changelog.

> Do we want to later differentiate between kinds of changes, e.g. features vs bugfix vs breaking changes etc.?

We can differentiate between those different types of changes, don't mind.

**Review by merveenoyan — APPROVED** [review]
LGTM 🌟

**Pull request merged**